tests/test_graphics_mem_safety.py

```diff
@@ -0,0 +1,65 @@
+"""Note: This tests are potential crashers (may result in segfaults)"""
+import pytest
+import imgui
+
+@pytest.fixture
+def context():
+    return imgui.create_context()
+
+@pytest.fixture
+def io():
+    # setup io
+    io = imgui.get_io()
+    io.delta_time = 1.0 / 60.0
+    io.display_size = 300, 300
+
+    # setup default font
+    io.fonts.get_tex_data_as_rgba32()
+    io.fonts.add_font_default()
+    io.fonts.texture_id = 42  # set any texture ID to avoid segfaults
+
+    return io
+
+def test_texture_id_int_reference(context, io):
+
+    # See issue #248 (https://github.com/pyimgui/pyimgui/issues/248)
+
+    texture_id = 0
+    for i in range(0,10000):
```

Comment:
I would say that this is a bit overkill, what about `range(0, 10000, 100)` or `range(0, 1000, 50)`

Reply:
Agree